core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -245,6 +246,11 @@ public static LoadTableResponse updateTable(Catalog catalog, TableIdentifier ide
         .build();
   }
 
+  public static void renameTable(Catalog catalog, RenameTableRequest request) {
+    request.validate();
```

Comment:
I don't think other methods in `CatalogHandlers` run `validate`. I think that should be done before passing it in here.

Reply:
Oh good point they don't. Since we're building them via the builder, it's not really necessary. Plus validate is called in the constructor anyway.